clients/privacy-center/packages/fides-consent/src/lib/cookie.ts

```diff
@@ -57,32 +129,58 @@ export const makeDefaults = ({
   return defaults;
 };
 
-export const getConsentCookie = (
-  defaults: CookieKeyConsent
-): CookieKeyConsent => {
+export const getOrCreateFidesCookie = (
```

Comment:
this func would benefit from a good ol' docstring. I think it'd be useful to clarify that `defaults` is _consent_ defaults, and not defaults for any of the other values in the cookie
```typescript
/** 
 * A docstring here will show up in IDEs when using this func!
 */ 
export const getOrCreateFidesCookie = (
```

Reply:
Ooh, I wrote *so many* comments but not the one that truly matters! Good call out

Reply:
If `getOrMake` describes it better to you, I think that's perfect